Core/tests/ExpressionParser2.cpp

```diff
@@ -1273,6 +1310,98 @@ TEST_CASE("ExpressionParser2", "[common][events]") {
       REQUIRE(validator.GetErrors()[0]->GetStartPosition() == 0);
       REQUIRE(validator.GetErrors()[0]->GetEndPosition() == 14);
     }
+    SECTION("the object doesn't exist") {
```

Comment:
> Question because I don't remember: do we have tests for failure cases (the object name does not exist as an object or as a group?)

Do you mean these cases?

Reply:
Oh right, not sure I missed this 🤦🏻‍♂️👍